common/web/lm-worker/build.sh

```diff
@@ -104,9 +91,9 @@ fi
 # of typescript, we need to avoid this!
 # TODO: we should try and rework this to avoid the need to manually wrap
 
-if builder_has_action clean || builder_has_action build; then
+if builder_has_action clean || builder_has_action build >/dev/null; then
```

Comment:
Adding a `>/dev/null` here intentionally?  I'm not clear on why it's needed here but not in many other places.

Reply:
Yeah, because the `builder_has_action build` was triggering a build status message here that was confusing -- because we have to do the clean even for build, see comments in previous lines... I'm not that happy with this, and a `--quiet` option may be better, but this will do for now I think.